com.unity.ml-agents/Runtime/Sensors/RayPerceptionSensorComponent2D.cs

```diff
@@ -14,7 +22,11 @@ public class RayPerceptionSensorComponent2D : RayPerceptionSensorComponentBase
         public RayPerceptionSensorComponent2D()
         {
             // Set to the 2D defaults (just in case they ever diverge).
+#if MLA_UNITY_PHYSICS2D_MODULE_ENABLED
```

Comment:
Can remove this whole section now that `k_PhysicsDefaultLayers` is in RayPerceptionSensorComponentBase.cs